Formula/apt.rb

```diff
@@ -0,0 +1,120 @@
+  sha256 "c946a126263b3ea51a15216689602367c0da95d6878b76dc891ea335d87bd936"
+  license "GPL-2.0-or-later"
+
+  livecheck do
+    url "https://deb.debian.org/debian/pool/main/a/apt/"
+    regex(/href=.*?apt[._-]v?(\d+(?:\.\d+)+)\.t/i)
+  end
+
+  depends_on "cmake" => :build
+  depends_on "docbook" => :build
+  depends_on "docbook-xsl" => :build
+  depends_on "doxygen" => :build
+  depends_on "googletest" => :build
+  depends_on "libxslt" => :build
+  depends_on "po4a" => :build
+  depends_on "w3m" => :build
+
+  depends_on "berkeley-db"
+  depends_on "bzip2"
+  depends_on "dpkg"
+  depends_on "gcc"
+  depends_on "gettext"
+  depends_on "gnupg"
+  depends_on "gnutls"
+  depends_on :linux
+  depends_on "lz4"
+  depends_on "perl"
+  depends_on "xxhash"
+  depends_on "zlib"
+
+  fails_with gcc: "5"
+
+  resource "SGMLS" do
+    url "https://cpan.metacpan.org/authors/id/R/RA/RAAB/SGMLSpm-1.1.tar.gz"
+    sha256 "550c9245291c8df2242f7e88f7921a0f636c7eec92c644418e7d89cfea70b2bd"
+  end
+
+  resource "triehash" do
+    url "https://github.com/julian-klode/triehash/archive/v0.3.tar.gz"
+    sha256 "289a0966c02c2008cd263d3913a8e3c84c97b8ded3e08373d63a382c71d2199c"
+  end
+
+  resource "Unicode::GCString" do
+    url "https://cpan.metacpan.org/authors/id/N/NE/NEZUMI/Unicode-LineBreak-2019.001.tar.gz"
+    sha256 "486762e4cacddcc77b13989f979a029f84630b8175e7fef17989e157d4b6318a"
+  end
+
+  resource "Locale::gettext" do
+    url "https://cpan.metacpan.org/authors/id/P/PV/PVANDRY/gettext-1.07.tar.gz"
+    sha256 "909d47954697e7c04218f972915b787bd1244d75e3bd01620bc167d5bbc49c15"
+  end
+
+  resource "Term::ReadKey" do
+    url "https://cpan.metacpan.org/authors/id/J/JS/JSTOWE/TermReadKey-2.38.tar.gz"
+    sha256 "5a645878dc570ac33661581fbb090ff24ebce17d43ea53fd22e105a856a47290"
+  end
+
+  resource "Text::WrapI18N" do
+    url "https://cpan.metacpan.org/authors/id/K/KU/KUBOTA/Text-WrapI18N-0.06.tar.gz"
+    sha256 "4bd29a17f0c2c792d12c1005b3c276f2ab0fae39c00859ae1741d7941846a488"
+  end
+
+  resource "YAML::Tiny" do
+    url "https://cpan.metacpan.org/authors/id/E/ET/ETHER/YAML-Tiny-1.73.tar.gz"
+    sha256 "bc315fa12e8f1e3ee5e2f430d90b708a5dc7e47c867dba8dce3a6b8fbe257744"
+  end
+
+  resource "Module::Build" do
+    url "https://cpan.metacpan.org/authors/id/L/LE/LEONT/Module-Build-0.4222.tar.gz"
+    sha256 "e74b45d9a74736472b74830599cec0d1123f992760f9cd97104f94bee800b160"
+  end
+
+  resource "Pod::Parser" do
+    url "https://cpan.metacpan.org/authors/id/M/MA/MAREKR/Pod-Parser-1.63.tar.gz"
+    sha256 "dbe0b56129975b2f83a02841e8e0ed47be80f060686c66ea37e529d97aa70ccd"
+  end
+
+  def install
+    # Find our docbook catalog
+    ENV["XML_CATALOG_FILES"] = "#{etc}/xml/catalog"
+
+    ENV.prepend_create_path "PERL5LIB", buildpath/"lib/perl5"
+    ENV.prepend_path "PERL5LIB", buildpath/"lib"
+    ENV.prepend_path "PATH", bin
+
+    resource("triehash").stage do
+      bin.install "triehash.pl" => "triehash"
```

Comment:
Last thing would be to confirm if this can be made build-only, which will allow us to make Perl build-only:
```suggestion
    ENV.prepend_path "PATH", buildpath/"bin"

    resource("triehash").stage do
      (buildpath/"bin").install "triehash.pl" => "triehash"
```